__local/grc20_tokens/g1er355fkjksqpdtwmhf5penwa82p0rhqxkkyhk5/register_gnodev/register_gnodev.gno

```diff
@@ -0,0 +1,233 @@
+package register_gnodev
+
+import (
+	pusers "gno.land/p/demo/users"
+
+	"gno.land/r/onbloc/bar"
+	"gno.land/r/onbloc/baz"
+	"gno.land/r/onbloc/foo"
+
+	"gno.land/r/onbloc/obl"
+	"gno.land/r/onbloc/qux"
+	"gno.land/r/onbloc/usdc"
+
+	"gno.land/r/demo/foo20"
+	"gno.land/r/demo/wugnot"
+
+	"gno.land/r/gnoswap/v1/gns"
+
+	cp "gno.land/r/gnoswap/v1/community_pool"
+	gs "gno.land/r/gnoswap/v1/gov/staker"
+	lp "gno.land/r/gnoswap/v1/launchpad"
+	pl "gno.land/r/gnoswap/v1/pool"
+	pf "gno.land/r/gnoswap/v1/protocol_fee"
+	rr "gno.land/r/gnoswap/v1/router"
+	sr "gno.land/r/gnoswap/v1/staker"
+)
+
+type FooToken struct{}
+
+func (FooToken) Transfer() func(to pusers.AddressOrName, amount uint64) {
+	return foo.Transfer
+}
+func (FooToken) TransferFrom() func(from, to pusers.AddressOrName, amount uint64) {
+	return foo.TransferFrom
+}
+func (FooToken) BalanceOf() func(owner pusers.AddressOrName) uint64 {
+	return foo.BalanceOf
+}
+func (FooToken) Approve() func(spender pusers.AddressOrName, amount uint64) {
+	return foo.Approve
+}
```

Comment:
you can probably simplify this file by making a token wrapper and then wrapping once per token,

Reply:
We remove this `register_gnodev` realm.
> FYI, it's something that we used to support multiple grc20 token before `grc20reg` comes in.